pygfx/renderers/wgpu/backgroundshader.py

```diff
@@ -161,3 +166,114 @@ def code_fragment(self):
             return out;
         }
         """
+
+
+@register_wgpu_render_function(Background, BackgroundSkyboxMaterial)
+class SkyboxShader(WorldObjectShader):
+
```

Comment:
Any particular reason why you implement a new shader instead of fixing the existing one?

Reply:
Err, the main reason is that I think the current implementation is a little complicated...
In the current implementation, the Skybox does not have a box like "entity" as it literally means. Its implementation process only exists in shaders and is procedure oriented.

In the class design, Skybox, as a special world object, can be added to the scene. It has a box entity. The only difference from ordinary world objects is that its position is fixed relative to the camera (in other words, it always follows and surrounds the camera). That is,, although the Skybox cannot be translated, other transformations such as rotation and scaling can be applied to Skybox objects.

Typical scenarios include:
1. Adjust the rotation angle and orientation of skybox:
 For example: `background.rotation.multiply(rot)`
 (Er, I'm not sure if this is the problem that the `up` property was designed to solve , 😅 )

2. Background follows the overall transformation of the scene:
 For example:  when `scene.rotation.multiply(rot)`,  The background will rotate with the scene

I think it may also be helpful to add other Skybox texture mapping modes, (`EquirectangularMapping`, etc.), The current implementation maybe has some "CubeTexture specific".

Of course, if you prefer the current implementation method, I would be happy to fix it on the current implementation. 😄